doc/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
-cmake_minimum_required(VERSION 3.13.1)
+cmake_minimum_required(VERSION 3.15)
```

Comment:
Let's wait with bumping the minimum required version to 3.15 for doc builds until we bump it for whole of Zephyr.
We are planning to increase the minimum version in general for improved toolchain handling.

It is correct that `ADDITIONAL_CLEAN_FILES` is new from 3.15, but it is ignored on older CMakes, so using this is not sufficient to request a higher CMake, imho.

Of course users of CMake <3.15 will just have to live with not getting all files cleaned.

Reply:
Fair enough, I thought it was required to work with `ADDITIONAL_CLEAN_FILES`